src/chapters/04/main.tex

```diff
@@ -25,59 +25,63 @@ \section{Theory}\label{sec:theory}
 mathematically using a differential equation which is a way of relating the rate
 of change of a system to the state of the system itself.
 
-In general if we are interested in some variable \(x\) over time \(t\) the
-differential function equation will be of the form:
+In general the objects of interest are the variable \(x\) over time \(t\), and
+the rate at which \(x\) changes with \(t\), its derivative $\frac{dx}{dt}$.
+The differential function equation describing this will be of the form:
 
 \begin{equation}
     \frac{dx}{dt} = f(x)
 \end{equation}
 
-For some function \(f\).
-In our case,
-if we denote the number of infected individuals as \(I\) where we implicitly
-mean that \(I\) is a function of time: \(I=I(t)\) and the rate at which
-individuals recover by \(\alpha\) then the differential equation
-that describes the above situation is:
+for some function \(f\).
+In this case,
+the number of infected individuals will be denoted as \(I\), which will
+implicitly mean that \(I\) is a function of time: \(I=I(t)\), and the rate at
+which individuals recover by will be denoted by \(\alpha\), then the
+differential equation that describes the above situation is:
 
 \begin{equation}
     \frac{dI}{dt} = -\alpha I
 \end{equation}
 
 Finding a solution to this differential equation means finding an expression for
-\(I\) that when differentiated gives \(- alpha I\).
+\(I\) that when differentiated gives \(- \alpha I\).
 
 In this particular case, one such function is:
 
 \begin{equation}
     I(t) = e ^ {-\alpha t}
 \end{equation}
 
-However, \(I(0) = 1\) whereas for our problem we know that at time \(t=0\) there
-are 100 infected individuals. Indeed a differential equation defines a family of
-solutions and we need to know some sort of initial (also referred to as
-boundary) condition to have the exact solution. Which in this case would be:
+% TODO Is it worth 'showing' that this is indeed a solution, e.g. \(\frac{dI}{dt} = -\alpha e ^ {-\alpha y} = -\alpha I). It's something our first year students need to be shown.
```

Comment:
Good question, my hunch is "no" because of:


```
> Concepts that are taught in a typical University mathematics course can be
 	  assumed but should include reference texts as **footnotes**.
``` 

So the concept of the DE is assumed to be known and we don't need to go in to the fact that a solution exists...

What do you think?

Reply:
Spoken with @geraintpalmer, agreed that it's essentially just a sentence so going to put it in.